docs/en/status-communities/understand-token-requirements-in-communities.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 490
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Understand token requirements in communities
+
+<!--
+image
+-->
+
+In Status, there are two kinds of communities: public and private. If you wish to join a private community, you need to meet the community owner token requirements. To maintain access to a private community, hold the necessary tokens at all times.
+
+Community owners can configure token requirements for their communities. To join a community you need to meet its token requirements. For example, a community owner can set up 10 SNts as a requirement to join a community. So you need to have a minimum of 10 SNTs to join.
+
+The Status app classifies tokens into three different categories: assets, collectibles, and ENS (Ethereum Name Service) names. Community owners can configure any combination of these tokens as a requirement to join their communities.
+
+!!! note
+    When you join a private community, the token requirements are not a form of payment or transfer. They are simply used for verification purposes, and your tokens remain in your possession
+
+## You meet the requirements
+
+Before joining a private community, the Status app automatically checks if you meet the community requirements. If you meet the requirements, the option to join the community is available.
+
+!!! note
+    If the community owner has set [manual approval][set-up-your-community-s-approval-options] option for their community, you still need the owner's approval to join.
+
+## You don't meet the token requirements
+
+When you don't meet the token requirements to join a community in the Status app, the request button will be unavailable and greyed out. However, you can acquire the necessary tokens to meet the requirements to join the community. Meeting the token requirements means owning the tokens and the required amount of tokens.
```

Comment:
- 'When' -> 'if'? Perhaps it'd make sense to reword this to 'If the request button is greyed out and unavailable, it means that <...>'.
- 'To apply to join the community, you need to have the required amount of tokens'?